sphinx/texinputs/sphinxlatexobjects.sty

```diff
@@ -146,6 +146,21 @@
   \item[{#1\sphinxcode{(}\py@sigparams{#2}{#3}\strut}]
   \pysigadjustitemsep
 }
+\newcommand{\pysigwithmultilineargsret}[3]{%
+  % render each argument on its own line
+  \item[#1\sphinxcode{(}\strut]
+
```

Comment:
```suggestion
  \leavevmode\par\nopagebreak
  % this relies on \pysigstartsignatures having set \parskip to zero
```

To avoid a page break occurring right above first function argument.